FMS/Pages/Shared/_PaginationNav.cshtml

```diff
@@ -37,7 +37,7 @@
                 {
                     <li class="page-item active" aria-current="page">
                         <span class="page-link">
-                            @index.ToString() <span class="sr-only">(current)</span>
```

Comment:
This is important for [accessibility](https://getbootstrap.com/docs/5.3/getting-started/accessibility/#visually-hidden-content). Restore this and use the `visually-hidden` class (instead of `sr-only`).

Reply:
OK, I changed this because the (current) text was messing up the pagination, especially in compact browser mode